certified-connectors/PDFco/readme.md

```diff
@@ -0,0 +1,15 @@
+# PDF.co
+PDF.co extracts data from pdf, splits pdf, merges pdf, converts PDF, translates PDF, adds and removes security options and other PDF tools. PDF.co API platform is trusted by 50,000+ users. It's Secure and Scalable and powered by AI, ML and OCR.
+
+# Pre-Requisites
+All PDF.co endpoints use API Key for request authentication. This API Key can be obtained by [signing-up](https://app.pdf.co/signup) at PDF.co.
+
+# API Documentation
+The API documentation can be found at https://apidocs.pdf.co
+
```

Comment:
@hiren-byte please format the readme.md file according to [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md?plain=1), all of the required headers should be present.